content_scripts/desktopRenderer.js

```diff
@@ -158,6 +158,11 @@ import { UNIFORMS, MAX_VIEW_FRUSTUMS } from '../../src/gui/ViewFrustum.js';
                     gltf.traverse(obj => {
                         if (obj.type === 'Mesh' && obj.material) {
                             obj.oldMaterial = greyMaterial;
+
+                            // to improve performance on mobile devices, switch to simpler material (has very large effect)
+                            if (!realityEditor.device.environment.isDesktop() && typeof obj.originalMaterial !== 'undefined') {
+                                obj.material = obj.originalMaterial;
```

Comment:
I think this could also start with obj.material.dispose() to free the resources from the previous material